cmake/modules/AddSwift.cmake

```diff
@@ -110,14 +121,15 @@ function(_add_variant_c_compile_link_flags)
   endif()
 
   if(CFLAGS_ENABLE_LTO)
-    list(APPEND result "-flto")
+    _compute_lto_flag(${CFLAGS_ENABLE_LTO} _lto_flag_out)
+    list(APPEND result "${_lto_flag_out}")
   endif()
```

Comment:
You could move the if statement into `_compute_lto_flag` (but then make sure to remove the double quotes from the append command).